index.js

```diff
@@ -13,6 +14,7 @@ mongoose.connect(config.mongoUrl);
 var app = express();
 app.use(express.static(path.join(__dirname, 'public')));
 app.use(urlencoded({ extended: true }));
+app.use(bodyParser.json());
```

Comment:
Is this necessary for every request, or can this middleware just be added for the Authy webhook route?